csv/src/fs2/data/csv/package.scala

```diff
@@ -40,4 +43,11 @@ package object csv {
   def noHeaders[F[_]](implicit F: ApplicativeError[F, Throwable]): Pipe[F, NonEmptyList[String], CsvRow[Nothing]] =
     CsvRowParser.pipe[F, Nothing](false)
 
+  def decode[F[_], R](implicit F: ApplicativeError[F, Throwable], R: RowDecoder[R]): Pipe[F, NonEmptyList[String], R] =
```

Comment:
I wonder whether it would make sense to additionally provide `attempt`-versions like
```scala
def attemptDecode[F[_], R](implicit F: Applicative[F], R: RowDecoder[R]): Pipe[F, NonEmptyList[String], DecoderResult[R]]
```
Of course the implementation is extremely simple, but this would be the place where I as a user of the library would look first and be happy to get a solution without diving into details.

Reply:
True, I will add it.